src/app/onboarding/review/page.tsx

```diff
@@ -0,0 +1,227 @@
+'use client';
+
+import { useState } from 'react';
+import Link from 'next/link';
+import { useRouter } from 'next/navigation';
+import { BigBlueAsyncButton, BigLinkButton } from '@/components/Buttons';
+import Icon from '@/components/Icon';
+import COLORS from '@/styles/colors';
+import { Card, Flex } from '@/styles/containers';
+import { H1Centered, H2, H4, P } from '@/styles/text';
+import { formatTruthy, getUTCDate, parseDate } from '@/utils/helpers';
+import { useGuardedOnboarding, useOnboardingNavigation } from '@/utils/hooks';
+import * as Styles from '../styles';
+
+function EditButton({ href }: { href: string }) {
+  return (
+    <Link href={href}>
+      <Flex $gap="5px">
+        <Styles.EditText>Edit</Styles.EditText>
+        <Icon type="edit" />
+      </Flex>
+    </Link>
+  );
+}
+
+export default function Page() {
+  const onboarding = useGuardedOnboarding();
+  const [errorMessage, setErrorMessage] = useState('');
+  const { flowAt, backlinkHref } = useOnboardingNavigation();
+  const { push } = useRouter();
+
+  // triggers on clicking submit
+  const onSubmit = async () => {
+    try {
+      await Promise.all([push('/onboarding-complete'), onboarding.flushData()]);
+    } catch (error) {
+      const msg = error instanceof Error ? error.message : String(error);
+      setErrorMessage(msg);
+    }
+  };
+
+  return (
+    <Card>
+      <Flex $direction="column" $gap="30px">
+        <Styles.BackLink href={backlinkHref}>
```

Comment:
`BackLinkButton` is used to trigger an on click effect to call `ebbTo`, which first checks if the current form is valid before redirecting to the specified URL. This is required for most onboarding pages.

We use `BackLink` for review and basic information because there's no need to check if that form is invalid or not.
- There's no form to fill out in review page
- There's no progress bar in the roles page, so there's no issue of invalid basic info -> roles -> review

Reply:
gotcha, ty for the clarification!